FILEIO.md

```diff
@@ -0,0 +1,44 @@
+# Arkouda File Support
+
+*Please Note: This file is being developed in conjunction with updates to our file I/O system. Information is being omitted until updates on each section are completed to avoid confusion.*
+
+## Table of Contents
+1. [File Types](#filetypes)
+2. [HDF5](#hdf)
+   1. [Multidimensional Objects](#multidim)
+
+## Supported File Types
+Arkouda currently supports 2 file types (listed below). When storing data, the way the data is stored may vary. This file will detail out the "schema" each file type is expected to follow. If your file does not follow the detailed "schema", please try utilization of our `import`/`export` tools. *Please Note: utilization of these tools is dependent on the size of the data because they only run on the client.*
```

Comment:
Right in the middle of this paragraph you have "please try utilization of our import/export tools" I think this would flow better as "please try using our import/export tools"